invenio_opendata/testsuite/data/alice/alice-derived-datasets.xml

```diff
@@ -1,6 +1,6 @@
 <collection>
   <record>
-    <controlfield tag="001">1102</controlfield>
+    <controlfield tag="001">1120</controlfield>
     <datafield tag="024" ind1="7" ind2=" ">
       <subfield code="a">10.7483/OPENDATA.ALICE.Y62S.E7UR</subfield>
       <subfield code="2">DOI</subfield>
```

Comment:
@AnxhelaDani @pherterich @suenjedt Note that this change may need to amend the DOI resolution.